packages/noya-state/src/reducers/pageReducer.ts

```diff
@@ -80,7 +85,9 @@ export function pageReducer(
 
           Layers.visit(page, (layer) => {
             layer.do_objectID = uuid();
+
             if (layer.style) layer.style.do_objectID = uuid();
+            if (layer._class === 'symbolMaster') layer.symbolID = uuid();
```

Comment:
We also want to update any instances within this page that reference the symbolMaster. I might do this in 2 passes: first we figure out all the new symbolIDs, then we assign them.

```ts
const mappedSymbolIDs: Record<string, string> = {}

page.layers.filter(Layers.isSymbolMaster).forEach(symbolMaster => {
  mappedSymbolIDs[symbolMaster.symbolID] = uuid()
})

Layers.visit(page, (layer) => {
  // ...
  if (layer._class === 'symbolMaster' 
      || (layer._class === 'symbolInstance' && layer.symbolID in mappedSymbolIDs)) { 
    layer.symbolID = mappedSymbolIDs[layer.symbolID]
  }
})
```